lmcache/storage_backend/local_backend.py

```diff
@@ -114,7 +114,7 @@ def put_nonblocking(self, key, kv_chunk):
         # Obtain keys to evict
         self.update_lock.acquire()
         evict_keys, put_status = self.evictor.update_on_put(
-            self.dict, kv_chunk)
+            self.dict, self.mpool.size_per_chunk)
```

Comment:
Why using `self.mpool.size_per_chunk` instead of `self.evictor.get_size(kv_chunk)` here?

What if the KV cache is not a "full chunk"?

Reply:
A chunk always counts as a full chunk. This is because a full chunk is always allocated in mempool. Otherwise, there's a misalignment between the evictor's `current_cache_size` and the actual allocated cache size in mempool.

Reply:
Oh I see (I thought it was disk backend because github collapsed the code)